frontend/src/components/ModalTravel.vue

```diff
@@ -0,0 +1,201 @@
+              {{ booker.name }} ({{ booker.type }})
+            </option>
+          </select>
+        </div>
+        <div class="mb-4">
+          <label class="block text-gray-700 text-sm font-bold mb-2" for="book">Reservation automatique</label>
+          <input
+            class="form-checkbox leading-tight focus:outline-none"
+            id="book"
+            type="checkbox"
+            v-model="book"
+          />
+        </div>
+        <div class="mb-4">
+          <label class="block text-gray-700 text-sm font-bold mb-2" for="notifier">Notifier</label>
+          <select
+            class="shadow appearance-none border rounded w-full py-2 px-3 text-gray-700 leading-tight focus:outline-none"
+            id="notifier"
+            type="text"
+            v-model="notifier"
+          >
+            <option disabled value="">Select a notifier</option>
+            <option v-for="notifier in notifiers" v-bind:key="notifier.id" v-bind:value="notifier.id">
+              {{ notifier.name }}
+            </option>
+          </select>
+        </div>
+      </form>
+    </template>
+
+    <template v-slot:footer>
+      <button class="px-4 bg-transparent p-3 rounded text-blue-500 hover:bg-gray-100 hover:text-blue-700 mr-2" @click="close">Cancel</button>
+      <button
+        class="bg-blue-500 hover:bg-blue-700 text-white font-bold py-2 px-4 rounded focus:outline-none"
+        type="button"
+        v-on:click="submitTravel"
+      >Lancer la recherche</button>
+    </template>
+  </Modal>
+</template>
+
+<script>
+import Autocomplete from 'vuejs-auto-complete'
+import Modal from './Modal.vue'
+import store from '../store'
+
+export default {
+  name: 'App',
+  components: {
+    Autocomplete,
+    Modal,
+  },
+  data() {
+    return {
+      from: null,
+      to: null,
+      date: new Date(),
+      minHour: null,
+      maxHour: null,
+      notifier: null,
+      booker: null,
+      book: true,
+      autocompleteEndpoint: `${process.env.API_URL}/stations/autocomplete?uc=fr-FR&searchField=origin&searchTerm=`
+    }
+  },
+  computed: {
+    bookers() { return store.state.bookers },
+    notifiers() { return store.state.notifiers },
+  },
+  methods: {
+    formatAutocomplete (results) {
+      return results.filter((result) => {
+        return result.category === 'station' && (result.type === 'G' || result.type === 'L')
+      }).map(r => ({ label: r.label, id: r.rrCode }))
+    },
+    autocompleteSelected (key) {
+      return (value) => {
+        this[key] = value.rrCode
+        return value.label
+      }
+    },
+    async submitTravel () {
+      const res = await fetch(`${process.env.API_URL}/travels`, {
+        method: 'POST',
+        headers: {
+          'Accept': 'application/json',
+          'Content-Type': 'application/json'
+        },
+        body: JSON.stringify({
+          from: this.from,
+          to: this.to,
+          date: new Date(this.date.toDateString() + ' UTC'),
```

Comment:
Since `date` is a string in the db, could we send it with the right format here?